src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlSecurityTokenHandler.cs

```diff
@@ -804,6 +804,49 @@ public override SecurityToken ReadToken(XmlReader reader, TokenValidationParamet
             throw LogHelper.LogExceptionMessage(new NotSupportedException(LogMessages.IDX11950));
         }
 
+        /// <summary>
```

Comment:
Should these be in SamlSecurityTokenHandler.Validation.Internal.cs?

Reply:
Moved to .ReadToken, like we did with JWT.